rpcs3/rpcs3_version.h

```diff
@@ -5,5 +5,7 @@
 
 namespace rpcs3
 {
+	std::string get_branch();
+
 	extern const utils::version version;
-}
+}
```

Comment:
Don't remove newlines at the end of file

Reply:
Alright , will keep an eye out for that in future.